string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,51 @@
+/* exported titleCase */
+function titleCase(string) {
+  var newString = '';
+  var partString = '';
+  partString = partString + string.charAt(0).toUpperCase();
+  for (var i = 1; i <= string.length; i++) {
+    if ((string[i] !== ' ') & (i !== string.length)) {
+      partString = partString + string[i];
+    } else {
+      if (partString.length > 3) {
+        partString = partString.charAt(0).toUpperCase() + partString.slice(1);
```

Comment:
This isn't the ideal way to account for those connecting words. Now you've made it so no three letter words can get capitalized. A huge collection of words such as car, cat, bow, etc would like a word with you about your decision.